### PR TITLE
Update frequently-asked-questions.md

### PR DESCRIPTION
**Pull request by justinegeffen**
Update based on: https://github.com/mattermost/docs/pull/4176

### CODE REVIEW

**Comment by amyblais** [comment]
I'm assigning this to Carrie (Tech writer) but let me know if you need help.

File: contributors/onboarding/frequently-asked-questions.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ Email [hr@mattermost.com](mailto:hr@mattermost.com) with any questions or concer
 
 ## I’d like to take some time off. What’s the process for that?
 
-* Discuss with your manager the dates you plan to take off, and decide who will be your backup.
+* Discuss with your manager the dates you plan to take off and decide who will be your backup.
```
Comment (cwarnermm) [inline contributors/onboarding/frequently-asked-questions.md]:
```suggestion
* Discuss the dates you plan to take off with your manager, and decide who will be your backup.
```

File: contributors/onboarding/frequently-asked-questions.md [hunk 2]
```diff
@@ -74,28 +74,36 @@ Whenever possible, communicate using Mattermost. Email is typically reserved for
...
+* [Stand-up](https://community-daily.mattermost.com/private-core/channels/stand-up) **- Share availability updates, such as "Going to the dentist, back shortly"**
+* [UX Design](https://community.mattermost.com/core/channels/ux-design) **- Questions and discussion about product design**
+* [Welcome](https://community.mattermost.com/private-core/channels/welcome) **- Where new staff members are introduced on their first day**
 
 ## What does 0/5 mean?
 
 0/5 means you are not very committed or attached to an outcome. 5/5 indicates full confidence and that you would stake your reputation on the opinion. More about these and other terms can be found [here](https://docs.mattermost.com/process/training.html#terminology).
 
-## I am planning to move, what do I need to know? 
+## I am planning to move, what do I need to know?
```
Comment (cwarnermm) [inline contributors/onboarding/frequently-asked-questions.md]:
```suggestion
## I am planning to move. What do I need to know?
```

**Review by cwarnermm — CHANGES_REQUESTED** [review]
Minor suggestions provided inline. Very good read overall!

**Review by cwarnermm — APPROVED** [review]
Looks great! Thank you!

**Comment by justinegeffen** [comment]
I'm merging this as there are no major content changes. @natalie-hub if there are any issues please let me know and I can open a new PR. :)

**Pull request merged**